engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/content-hosts.module.js

```diff
@@ -265,7 +265,7 @@ angular.module('Bastion.content-hosts').config(['$stateProvider', function ($sta
         template: '<div ui-view></div>'
     })
     .state('content-hosts.details.errata.index', {
-        url: '/errata?search',
+        url: '/errata?getSearch',
```

Comment:
would it make sense to use the same search param in all details nutupane?  A param called 'detailsSearch' perhaps?